internal/statestore/redis.go

```diff
@@ -545,15 +549,9 @@ func handleConnectionClose(conn *redis.Conn) {
 	}
 }
 
-// TODO: add cache the backoff object
-func (rb *redisBackend) createConstantBackoffStrat() backoff.BackOff {
-	backoffStrat := backoff.NewConstantBackOff(rb.cfg.GetDuration("backoff.initialInterval"))
-	return backoff.BackOff(backoffStrat)
-}
-
 // TODO: add cache the backoff object
 // nolint: unused
-func (rb *redisBackend) createExponentialBackoffStrat() backoff.BackOff {
+func (rb *redisBackend) newExponentialBackoffStrategy() backoff.BackOff {
```

Comment:
ah .. the PR didnt show that you had another such helper. in that case, you might keep the other one too with newConstantBackoffStrategy()